File: Assignment11/program4.c
/////////////////////////////////////////////////////////////////// 
//
// File name : program4.c
// Description : Pattern
// input : 5
// Output :# 1 * # 2 * # 3 * # 4 * # 5 * 
// Author : Adinath Santosh Pawar
// Date : 27/5/25
// 
///////////////////////////////////////////////////////////////////

#include<stdio.h>

void Pattern(int iNo)
{
    int i = 0;

    for(i = 1; i <= iNo; i++)
    {
        printf("# %d * ",i);
    }
}

int main()
{
    int iValue = 0;
    printf("Enter the value \n");
    scanf("%d",&iValue);

    Pattern(iValue);

    return 0;
}

File: Assignment48/program4.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program4.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 20/8/25           
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(String str)
    {
        for(int i = 0; i < str.length(); i++)
        {
            for(int j = 0; j < str.length(); j++)
            {
                if(i + j < str.length())
                {
                    System.out.print(str.charAt(j) + " ");
                }
            }
            System.out.println();
        }

        for(int i = 1; i < str.length(); i++)
        {
            for(int j = 0; j < str.length(); j++)
            {
                if(i >= j )
                {
                    System.out.print(str.charAt(j) + " ");
                }
            }
            System.out.println();
        }
    }
}

class program4
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);

       System.out.print("Enter the string ");
       String str = sobj.nextLine();

       pobj.Pattern(str);
    }
}